react/src/components/organisms/TabContainer/tab.js

```diff
@@ -31,15 +35,53 @@ class Tab extends React.Component {
             this.defaultSet = true;
           }
           return(
-            <button
-              className={tabClasses}
-              onClick={(e) => {
-                this.scrollIntoView(e.target);
-                setActiveTab(this.tabIdent, this.props.children)
-              }}
-            >
-            {this.props.title}
-            </button>
+            <li role="presentation" className={tabClasses}>
+              <button
+                role="tab"
+                id={this.tabIdent}
+                onClick={(e) => {
+                  this.scrollIntoView(e.target);
+                  setActiveTab(this.tabIdent, this.props.children)
+                }}
+                aria-selected={active}
+                tabIndex={!active ? -1 : 0}
+                aria-describedby={this.props.a11yID}
+
+
+    // THIS EVENT IS ONLY AVAILABLE TO THE ONFOCUS TAB.
+                onKeyDown = {(e) => {
+                  if (e.keyCode === 37 || e.keyCode === 39) {
+                    e.preventDefault();
+                    e.stopPropagation();
+
+                    if (this.tabIdent !== document.activeElement.id) {
```

Comment:
You can actually use context for this. This has also already been set to the variable active above:
```suggestion
                    if (!active) {
```

Reply:
With `!active`, the active tab doesn't always match the tab with focus :-(
Am I missing something?
<img width="940" alt="banners_and_alerts_and_storybook" src="https://user-images.githubusercontent.com/9633303/48211056-ae857d80-e346-11e8-9cd4-79004afb9b53.png">